torch/_torch_docs.py

```diff
@@ -110,6 +110,20 @@ def merge_dicts(*dicts):
     "tf32_note": """This operator supports :ref:`TensorFloat32<tf32_on_ampere>`."""
 }
 
+
+reproducibility_notes = {
+    "forward_reproducibility_note": """When using the CUDA backend, this operation may induce nondeterministic \
+behavior that cannot be switched off. Please see the notes on :doc:`/notes/randomness` for background.""",
+    "backward_reproducibility_note": """When using the CUDA backend, this operation may induce nondeterministic \
```

Comment:
Per offline discussion:

"This operation may produce nondeterministic gradients when given tensors on a CUDA device. See :doc:/notes/randomness for more information."